src/AltinnCore/Runtime/appsettings.json

```diff
@@ -42,7 +42,8 @@
     "ApiAuthenticationEndpoint": "http://localhost:5040/authentication/api/v1/",
     "ApiAuthenticationEndpointHost": "localhost:5040",
     "ApiAuthorizationEndpoint": "http://localhost:50500/authorization/api/v1/",
-    "ApiAuthorizationEndpointHost": "localhost:5050"
+    "ApiAuthorizationEndpointHost": "localhost:5050",
```

Comment:
This should also be included in runtimes values.yaml with correct hots for dev/staging/prod

Reply:
Fixed